src/AspNetCoreModuleV2/AspNetCore/Inc/aspnetcore_shim_config.h

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
I personally don't the name aspnetcore_shim_config. As you already renamed another one to request_handler_config, you can keep current one

Reply:
I prefer including shim in the name. Keeps it separate from the old aspnetcore_config and makes it clear that the config is smaller.